### PR TITLE
Form for User Inputs

### PR DESCRIPTION
**Pull request by zorazrr**
Add form to take user input, post to `/form-handler` endpoint and display the input. 
Screen Shot 2022-06-08 at 10 44 51 PM
Screen Shot 2022-06-08 at 10 45 03 PM

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String message = request.getParameter("message");
+        String email = request.getParameter("email");
+        String forfun = request.getParameter("forfun");
```
Comment (emilyzhang242) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:18]: Super nitpick: Common naming convention for javascript is camel case, so "forfun" in this case would be "forFun". This helps with readability!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -97,7 +97,59 @@ <h2>Projects</h2>
...
+              >Leave a message for me</label
+            >
+            <textarea
+              class="form-control"
+              name="message"
+              id="message"
+              rows="3"
+              placeholder="your message"
+              required
+            ></textarea>
```
Comment (emilyzhang242) [inline portfolio/src/main/webapp/index.html:124]:
```suggestion
            <label for="message" class="form-label">
                Leave a message for me
            </label>
            <textarea
              class="form-control"
              name="message"
              id="message"
              rows="3"
              placeholder="your message"
              required></textarea>
```

It's nice to keep the closing brackets together. This is probably more optional than strictly required, but helps with readability. Please apply this to all instances here and below.

**Review by emilyzhang242 — CHANGES_REQUESTED** [review]
Overall great job!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (zorazrr) [reply portfolio/src/main/webapp/index.html:124]: Hi Emily! I believe this is due to auto-formatting from using prettier as a pre-commit hook. Would you recommend getting rid of the hook? Or is there a better work-around?

Comment (emilyzhang242) [reply portfolio/src/main/webapp/index.html:124]: Hmm, is there a way to configure prettier somehow? It seems like it has configs but not sure if you have a way to use them: https://prettier.io/docs/en/configuration.html

if not, I guess it's not a huge deal, but just something to be aware of.

Comment (zorazrr) [reply portfolio/src/main/webapp/index.html]: I played around with prettier config a bit, but did not work out. I ended up getting rid of prettier pre-commit hook if that's ok (will use the editor's default formatting tool going forward). Will merge this PR.

**Pull request merged**